### DISCUSSION TITLE
Can not connect to IBMi.

### DISCUSSION THREAD

**Discussion by owenjpatrick** [codefori/vscode-ibmi#1934] [IBM i]
Can not connect to IBMi.  I have run the STRTCPSVR SERVER(*SSHD).  How do determine and change the PORT?

**Comment by pabloto** [comment]
Hi, by default ssh server run on port 22, you could check with netstat option 3 and see if ssh server is active and run on that port.

The ssh server configuration is here:

/QOpenSys/QIBM/ProdData/SC1/Openssh/etc/sshd_config (not sure of the correct case).
Here's you can check the configuration. Be careful on making changes.

Bye
